### PR TITLE
Roundstart Fixes

### PR DESCRIPTION
**Pull request by furrycactus**
Adds the shit that didn't come with my roundstart PR because Skull told me the map broke or something

Also fixes the wires in the surface shield room ( #5551 ), the surface level powernets ( #5550 ) and puts a sink in hydroponics.

### CODE REVIEW

**Comment by furrycactus** [comment]
Why in the GODDAMN DOES IT HAVE DOC'S PR IN HERE

**Comment by furrycactus** [comment]
WHY IS IT CONFLICTING

**Comment by furrycactus** [comment]
!review

File: code/modules/clothing/spacesuits/rig/suits/light.dm [hunk 1]
```diff
@@ -92,7 +92,8 @@
 		/obj/item/rig_module/voice,
 		/obj/item/rig_module/ai_container,
 		/obj/item/rig_module/datajack,
-		/obj/item/rig_module/self_destruct
+		/obj/item/rig_module/self_destruct,
+		/obj/item/rig_module/actuators
```
Comment (Arrow768) [inline code/modules/clothing/spacesuits/rig/suits/light.dm]: That looks like you are merging something from dev into master ?

Comment (furrycactus) [reply code/modules/clothing/spacesuits/rig/suits/light.dm]: I have no idea. I'm 99% sure that came with what Skull did.

Comment (furrycactus) [reply code/modules/clothing/spacesuits/rig/suits/light.dm]: Wait. Maybe that's because Doc's PR is in here for some reason too. God damnit.

**Comment by furrycactus** [comment]
Rebasing to master to try and fix this mess.

**Comment by furrycactus** [comment]
fuck it i'll just start over again this is driving me insane

**Pull request closed without merging**